06_create_cluster.sh

```diff
@@ -57,11 +60,16 @@ fi
 # Build a new release, based on the configured release, that overrides the
 # appropriate images built above.
 if [ -n "$IMAGE_OVERRIDES" ]; then
-    # This is consistently failing for me for some reason. It seems to be
-    # completing anyway though...
+    # This is what I would like to do, but it doesn't work
+#     oc adm release new -n ocp \
+#         --server https://api.ci.openshift.org \
+#         --from-release "$OPENSHIFT_RELEASE_IMAGE" \
+#         --to-image quay.io/cybertron/origin-release:v4.2 \
+#         $IMAGE_OVERRIDES || :
+    # This works in isolation but not when run as part of this script. :-/
```

Comment:
cc @derekhiggins @stbenjam - can we use this as an alternative to https://github.com/openshift-metal3/dev-scripts/pull/759 and https://github.com/openshift-metal3/dev-scripts/pull/719 ?

It seems like a common approach here for all potentially overridden container images would be good, and using a custom release probably aligns better with integration into openshift CI?

Reply:
Yeah I think this would be super helpful for testing a lot of new stuff.

Reply:
I've closed 719, not sure how I ended up with two patch doing similar stuff...

I wonder if we should include some way of using a local registry (similar to https://github.com/openshift-metal3/dev-scripts/pull/759) to save bandwidth? I'm currently looking into integrating into the openshift build system which may eventually provide an alternative.

Reply:
Last time I checked, there wasn't really a way to use a local registry with the installer. There's no way to tell it to use --tls-verify=false when retrieving images, so it barfs on the "insecure" registry.

I would dearly love it if we could start using a local registry though. The amount of bandwidth and time that I burn from repeatedly pulling the same images is absurd. :-/

Reply:
> Last time I checked, there wasn't really a way to use a local registry with the installer. There's no way to tell it to use --tls-verify=false when retrieving images, so it barfs on the "insecure" registry.
> 
> I would dearly love it if we could start using a local registry though. The amount of bandwidth and time that I burn from repeatedly pulling the same images is absurd. :-/

I couldn't find a way to do it either without hacking the code, see the PR I linked to above
https://github.com/openshift-metal3/dev-scripts/pull/759/files#diff-13094614d976ac6f9d0cc2f87e41f332R36